golang.mk

```diff
@@ -8,6 +8,7 @@
 
 have_yq=$(shell if [ -x "$(GOPATH)/bin/yq" ]; then echo "true"; else echo ""; fi)
 ifeq (,$(have_yq))
+    $(info yq was not found, installing it)
```

Comment:
Nit: Could you add a "log level prefix" to make this:

```
$(info INFO: yq was not found, installing it)
```

Reply:
That's a good idea